components/event-stream/org.wso2.carbon.event.stream.core/src/main/java/org/wso2/carbon/event/stream/core/internal/util/SampleEventGenerator.java

```diff
@@ -153,7 +153,13 @@ private static JsonObject createPropertyElement(List<Attribute> attributeList) {
 
         JsonObject innerObject = new JsonObject();
         for (Attribute attribute : attributeList) {
-            innerObject.addProperty(attribute.getName(), getSampleValue(attribute.getType()));
+            if (attribute.getType().name().toLowerCase().equals(EventStreamConstants.ATTR_TYPE_STRING)) {
+                innerObject.addProperty(attribute.getName(), getSampleValue(attribute.getType()));
+            } else if (attribute.getType().name().toLowerCase().equals(EventStreamConstants.ATTR_TYPE_BOOL)) {
+                innerObject.addProperty(attribute.getName(), Boolean.parseBoolean(getSampleValue(attribute.getType())));
+            } else {
+                innerObject.addProperty(attribute.getName(), Double.parseDouble(getSampleValue(attribute.getType())));
+            }
         }
```

Comment:
I did Double.parseDouble in else part because it will apply for int, float, double and long. 
Although it does a Double.parseDouble for int and long types it will show without decimal points in json object.